planner/property/task_type.go

```diff
@@ -40,7 +40,7 @@ func (t TaskType) String() string {
 		return "rootTask"
 	case CopSingleReadTaskType:
 		return "copSingleReadTask"
-	case CopDoubleReadTaskType:
+	case CopMultiReadTaskType:
 		return "copDoubleReadTask"
```

Comment:
This line is still DoubleRead. Should it be consistent?

Reply:
Yeah, let me change it to see whether it will influence the tests.

Reply:
I want to make sure that there are no tests caused by the changed codes